vscode-extension/src/extension.ts

```diff
@@ -21,10 +21,12 @@ export async function activate(context: ExtensionContext) {
 
   const coursierPath = path.join(context.extensionPath, "./coursier");
 
+  const serverVersion = workspace.getConfiguration("metals").get("serverVersion")
+
   const coursierArgs = [
     "launch",
     "-r", "bintray:scalameta/maven",
-    "org.scalameta:metals_2.12:0.1-SNAPSHOT",
+    `org.scalameta:metals_2.12:${serverVersion}`,
```

Comment:
I've tested this locally: the default currently returns an error (because there's no 0.1.0 artifact published yet), and changing it to `0.1-SNAPSHOT` works correctly.